js/js.tests/test/org/jetbrains/kotlin/generators/tests/GenerateJsTests.kt

```diff
@@ -85,6 +77,36 @@ fun main(args: Array<String>) {
                 model("codegen/boxError", targetBackend = TargetBackend.JS_IR)
             }
 
+            testClass<AbstractIrCodegenBoxWasmTest> {
+                model(
+                    "codegen/box", pattern = "^([^_](.+))\\.kt$", targetBackend = TargetBackend.WASM, excludeDirs = listOf(
+
+                        // JVM-specific
+                        "assert", "builtinStubMethods",
```

Comment:
Most of JVM specific tests already markaed by TARGET_BACKEND, there are some non JVM specific tests

Reply:
Removed it. Muted tests in place.